bucket/papercut.json

```diff
@@ -5,11 +5,17 @@
     "license": "Apache-2.0",
     "url": "https://github.com/ChangemakerStudios/Papercut/releases/download/5.5.1/Papercut.5.5.1.zip",
     "hash": "ec67c13414057cf9bf4e08002bc39ba148e2e79c7ec1d321a599f05374f7f26d",
+    "pre_install": [
+        "$file = 'rules.json'",
+        "if (!(Test-Path \"$persist_dir\\$file\")) {",
+         "    New-Item -Force -Path \"$dir\" -Name $file -Value '[]' | Out-Null",
+        "}"
+    ],
     "persist": [
         "Incoming",
-        "Logs"
+        "Logs",
+        "rules.json"
     ],
-    "bin": "Papercut.exe",
```

Comment:
Why was bin removed?

Reply:
It'a s GUI app - the shortcut is enough.

Reply:
I do not see a reason to remove binary. There are lots of manifests with applications which are graphical only and have binary.

Reply:
In this case, launching the shim gives an empty console window that stays on :(
I can't see any benefit from keeping the shim.